packages/expo-notifications/src/getExpoPushTokenAsync.ts

```diff
@@ -66,10 +60,10 @@ export default async function getExpoPushTokenAsync(
     );
   }
 
-  if (!experienceId && !projectId) {
+  if (!projectId) {
     throw new CodedError(
       'ERR_NOTIFICATIONS_NO_EXPERIENCE_ID',
-      "No experienceId or projectId found. If one or the other can't be inferred from the manifest (eg. in bare workflow), you have to pass one in yourself."
+      "No projectId found. If projectId can't be inferred from the manifest (eg. in bare workflow), you have to pass it in yourself."
```

Comment:
I would wrap `projectId` in quotes
```suggestion
      "No 'projectId' found. If 'projectId' can't be inferred from the manifest (eg. in bare workflow), you have to pass it in yourself."
```

Reply:
Sure, updated